diff --git a/node_modules/aegir/src/build/index.js b/node_modules/aegir/src/build/index.js
index 833fb1a..2845a27 100644
--- a/node_modules/aegir/src/build/index.js
+++ b/node_modules/aegir/src/build/index.js
@@ -40,7 +40,7 @@ const build = async (argv) => {
     {
       entryPoints: [entryPoint],
       bundle: true,
-      format: 'iife',
+      format: 'esm',
       conditions: ['production'],
       sourcemap: argv.bundlesize,
       minify: true,
